README.md

```diff
@@ -1 +1,19 @@
-# casbin-spring-security-starter
\ No newline at end of file
+# using casbin in SpringSecurity
+
+## inform
+    To simply display this example,
+    We will using default SpringSecurity login to save you from name and password.
+
+## DataBase 
+    Mysql
+    configue your database information in application.yml
+    create database casbin if you don't possess one 
+
+## Simple Example
+    using postMan send HttpRequest
```

Comment:
What's testAdapter, data and data1? Can you make it more meaningful? Like:

- alice can/cannot read data1
- bob can/cannot write data2